.github/CODEOWNERS

```
@@ -1,8 +1,4 @@
-* @Jigsaw-Code/outline-owners
+*                               @Jigsaw-Code/outline-dev
 
-/src/server_manager/messages/ @Jigsaw-Code/outline-strings-owners
-*.json                        @Jigsaw-Code/outline-strings-owners
-*.md                          @Jigsaw-Code/outline-strings-owners
-*.png                         @Jigsaw-Code/outline-strings-owners
-*.svg                         @Jigsaw-Code/outline-strings-owners
-LICENSE                       @Jigsaw-Code/outline-strings-owners
+/src/server_manager/model/      @fortuna 
+/src/shadowbox/model/           @fortuna 
```

For now I want to keep reviewing changes to the server.

```suggestion
/src/shadowbox/           @fortuna 
```